core/test-api/src/main/kotlin/testApi/testRunner/TestRunner.kt

```diff
@@ -31,19 +31,21 @@ abstract class AbstractTest<M : TestMethods, T : TestBuilder<M>, D : DokkaTestGe
     protected fun testFromData(
         configuration: DokkaConfigurationImpl,
         cleanupOutput: Boolean = true,
-        preserveOutputLocation: Boolean = false,
+        useOutputLocationFromConfig: Boolean = false,
```

Comment:
Having `cleanupOutput` and `preserveOutput` in one method is rather confusing as these are the two opposites.

Hopefully the new name represents the behaviour a little bit better